specs/relayer.md

```diff
@@ -0,0 +1,60 @@
+# Astria Sequencer Relayer
+
+The Astria Sequencer Relayer (referred to as relayer from here on out), reads
```

Comment:
maybe add a quick line explicitly explaining why we need the relayer; ie "data sequenced by the sequencer chain needs to be written to a data availability layer; this has been separated from the core sequencer node functionality into its own relayer program" or something like that